microservice/src/main/java/com/skytouch/model/ProductEntity.java

```diff
@@ -0,0 +1,112 @@
+package com.skytouch.model;
+
+
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@Table(name = "product")
+
+@NamedStoredProcedureQueries({
+        @NamedStoredProcedureQuery(name = "get_products",
+                procedureName = "get_products",
+                resultClasses = ProductEntity.class),
+        @NamedStoredProcedureQuery(name = "insert_product",
+                procedureName = "insert_product",
+                resultClasses = {ProductEntity.class},
+                parameters = {
+                        @StoredProcedureParameter(mode = ParameterMode.IN, name = "p_name", type = String.class),
+                        @StoredProcedureParameter(mode = ParameterMode.IN, name = "p_description", type = String.class),
+                        @StoredProcedureParameter(mode = ParameterMode.IN, name = "p_unit_price", type = Integer.class),
+                        @StoredProcedureParameter(mode = ParameterMode.IN, name = "p_quantity", type = Integer.class)
+                }
+        )
+})
+public class ProductEntity implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "name")
```

Comment:
if the column name has the same value as the class attribute, it is not necessary to use `@Column` annotation

Reply:
It's not necessary, but some variables do require it(unitPrice & quantityPerUnit), so to generate a standard, I decided to define all.